src/coreclr/minipal/Unix/CMakeLists.txt

```diff
@@ -9,3 +9,7 @@ add_library(coreclrminipal
 )
 
 target_link_libraries(coreclrminipal PRIVATE minipal)
+
+if(CLR_CMAKE_HOST_ANDROID)
+    install_clr(TARGETS coreclrminipal DESTINATIONS . sharedFramework)
```

Comment:
These are usually statically linked to other objects, not needed separately on their own. e.g. on iOS we don't call install target for these (internal details) objects.